crates/biome_js_type_info/tests/snapshots/infer_type_of_array_element.snap

```diff
@@ -0,0 +1,16 @@
+---
+source: crates/biome_js_type_info/tests/utils/mod.rs
+expression: content
+---
+```ts
+const array: Array<string> = [];
+
+```
+
+```
+array => instanceof "Array" {
+  params: unknown
```

Comment:
What are the `params` of an array? Not sure I understand this case

Reply:
Good catch, it was just a typo in the type formatter :) Fixed now.